.github/workflows/release.yml

```diff
@@ -18,7 +18,9 @@ jobs:
       - name: Install poetry
         run: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
       - name: Build distributions
-        run: poetry build
+        run: |
+          source $HOME/.poetry/env
```

Comment:
Hi @sduenas, I have tried and tested this. Thanks for creating the action, it worked perfectly fine except that it needs a small update as there are some issues with the current version. I have fixed it and tested it on fork branches.
https://github.com/vchrombie/poetry-python/commit/b6f096ee734611de5313cdd2641ae592b875cb84

tested with the new action file
https://github.com/vchrombie/release-tools/commit/c73d7a7aaae57a7ab7481259901caea25140d763
https://github.com/vchrombie/release-tools/actions/runs/716377830

Reply:
> Do you think is worth to maintain it in Bitergia (for example) or not?

We can either have this action setup in a repo or just add an extra line `echo "PATH=$HOME/.poetry/bin:$PATH" >> $GITHUB_ENV` right after installing poetry. I don't see any special advantage for both the methods.

Reply:
After some talking to @vchrombie, we decided to go for adding the env var to the workflow instead of creating a new action. As the installation of poetry is not complex, we prefer to avoid an extra dependency that we need to maintain.